yaml/nadav/‏‏‏‏yamel_exercise3_nadav .yaml

```diff
@@ -0,0 +1,38 @@
+ingredients:
+  base: &base
+    - 2 cups of broken lotus cookies
+    - 75g melted butter
+  cream: &cream
+    - 100g white chocolate
+    - 60ml milk
+    - 25g butter
+    - a pack of instant vanilla pudding
+    - 250g mascarpone cheese
+    - 250g cream cheese
+    -
+  fluff: &fluff
+    - 65g sugar
+    - 500ml sweet cream
+
+steps:
+  first: >
+    Mix the following ingredients in the base and put the mix in the bottom of a cake pan.
+    Put it in the freezer.
+  first_ing:
+  <<: *base
```

Comment:
missing indentation

Reply:
ur mama miss indentation

Reply:
10$ for a single slice??????
we are NOT made out of money